haiku-apps/midikeyboard/midikeyboard-1.0.0.recipe

```diff
@@ -0,0 +1,44 @@
+SUMMARY="An onscreen midi keyboard"
+DESCRIPTION="An awesome keyboard on the screen for playing music"
```

Comment:
You should come up with a little bit more. Try it out a bit, see what features it provides in the menus (right-clickk and the one in the bottom left).